mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/dependencies/DependencyResolver.java

```diff
@@ -60,8 +60,8 @@ private Dependency smartTestingProviderDependency() {
     }
 
     static class SurefireApiDependency extends Dependency {
-        private static final String GROUP_ID = "org.apache.maven.surefire";
-        private static final String ARTIFACT_ID = "surefire-api";
+        static final String GROUP_ID = "org.apache.maven.surefire";
+        static final String ARTIFACT_ID = "surefire-api";
```

Comment:
Do we really need to make it package-protected? Don't see any use of it in this PR.